.github/workflows/release.yml

```
@@ -0,0 +1,17 @@
+name: Release
+on:
+    push:
+        tags: ['*']
+permissions:
+    contents: write
+jobs:
+    release:
+        name: Release
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Create GitHub release
+              uses: docker://antonyurchenko/git-release:v5
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

What about some space?
```suggestion
name: Release

on:
    push:
        tags: ['*']

permissions:
    contents: write

jobs:
    release:
        name: Release
        runs-on: ubuntu-latest
        steps:
            - name: Checkout
              uses: actions/checkout@v2
            - name: Create GitHub release
              uses: docker://antonyurchenko/git-release:v5
              env:
                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

---

I only copied this from the Common Changelog website, but I agree more space make it better, I added it.